src/Symfony/Bridge/Twig/Resources/views/Form/foundation_5_layout.html.twig

```diff
@@ -269,7 +269,7 @@
     {% endif %}
     <label{% with { attr: label_attr } %}{{ block('attributes') }}{% endwith %}>
         {{ widget|raw }}
-        {{ translation_domain is same as(false) ? label : label|trans(label_translation_parameters, translation_domain) }}
+        {{- block('form_label_content') -}}
```

Comment:
To match the other form themes?
```suggestion
            {%- if label is not same as(false) -%}
                {{- block('form_label_content') -}}
            {%- endif -%}

Reply:
I don't know why someone would like a checkbox or radio without any label, but sure!